src/__tests__/components/exchangerCard.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import ExchangerCard from 'components/exchangerCard';
+
```

Comment:
- Please, fix the issue with not finding the module as this makes your test to fail


![image](https://github.com/fmanimashaun/cryptorank-app/assets/59397159/6b05725f-0589-4a11-8ff1-cefc3cd97fba)

Reply:
![image](https://github.com/fmanimashaun/cryptorank-app/assets/22054464/ab259433-dba9-4aed-9b7e-9ce1a000ff92)